Jenkinsfile

```diff
@@ -69,6 +69,9 @@ pipeline {
                         }
 
                         def testProfile = msg['update']['release']['dist_tag']
+                        if (testProfile == "epel10.0") {
```

Comment:
Let's not do it like this. If you look at the message, you can see that `msg['update']['release']['branch']` is actually `epel10` -- which is what we want. So we can use that value for the test profile name, if we are dealing with epel* updates.